### ISSUE TITLE
Internal Server Error (UPDATE)

### ISSUE THREAD

**Issue by LabSen** [SSilence/selfoss#534]
When I try to update my feeds, I have an Internal Server Error:

Internal Server Error

PDOStatement: Incorrect string value: '\xF0\x9F\x92\x8B' for column 'title' at row 1

debug log:

07-08-14 10:05:58 Debug ---
07-08-14 10:05:58 Debug start fetching source "Klicktoris (id: 107) 
07-08-14 10:05:58 Debug spout successfully loaded: spouts\rss\feed
07-08-14 10:05:58 Debug fetch content
07-08-14 10:05:58 Debug minimum date: 2013-07-08 10:05:58
07-08-14 10:05:58 Debug start item fetching
07-08-14 10:05:58 Debug start insertion of new item "babygirlssweetsurrender:

💋"
07-08-14 10:05:58 Debug item content sanitized
07-08-14 10:05:59 Debug icon generated: http://31.media.tumblr.com/avatar_f2f132a9919f_128.png

When i delet the source klicktoris works perfect. Probably a charset problem

**Comment by vegbrasil** [comment]
Hi guys, I'm having the same problem.

It happens with Twitter spout trying to fetch a tweet from `@htcvive`.

```
01-17-16 14:18:14 Debug start fetching source "@htcvive (id: 16) 
01-17-16 14:18:14 Debug spout successfully loaded: spouts\twitter\usertimeline
01-17-16 14:18:14 Debug fetch content
01-17-16 14:18:14 Debug minimum date: 2015-12-18 14:18:14
01-17-16 14:18:14 Debug start item fetching
01-17-16 14:18:14 Debug start insertion of new item "Engadget (RT HTC Vive):<br>Editor-in-Chief @Numeson demoing @htc Vive. {Follow us on Snapchat for more: gadgetsnaps 👻} #virtualreality <a href="https://t.co/IIIlEyoMeo" target="_blank">https://t.co/IIIlEyoMeo</a>"
01-17-16 14:18:14 Debug item content sanitized
01-17-16 14:18:14 Debug thumbnail generated: http://pbs.twimg.com/ext_tw_video_thumb/685938382954872833/pu/img/hQ_8ltr0Hj3JC6Y8.jpg
01-17-16 14:18:14 Debug icon generated: http://pbs.twimg.com/profile_images/655059892022022144/Pq3Q_1oU_normal.png
01-17-16 14:18:14 Error PDOStatement: Incorrect string value: '\xF0\x9F\x91\xBB} ...' for column 'title' at row 1
```

Following what @jtojnar said, I've set my database to `utf8mb4_general_ci` and changed the line `397` and `398` from `libs/f3/db/sql.php` to

`$options+=array(\PDO::MYSQL_ATTR_INIT_COMMAND=>'SET NAMES utf8mb4;');`

and the problem persists. I think the emoji is breaking the encoding? I'm missing something here? Really want to fix this.

Thanks!

**Comment by jtojnar** [comment]
Fixed in #813
